src/webnn_native/dml/GraphDML.cpp

```diff
@@ -490,6 +489,39 @@ namespace webnn_native { namespace dml {
         return {};
     }
 
+    ::dml::FusedActivation CreateFusedActivation(OperatorBase* activation) {
+        ::dml::FusedActivation dmlActivation;
+        if (activation == nullptr) {
+            return dmlActivation;
```

Comment:
What's the differentation from L.500` dmlActivation = ::dml::FusedActivation::None();`

Reply:
They both return ::dml::FusedActivation::None(). I will update codes here. Thanks!